apps/condo/domains/ticket/utils/unit.js

```diff
@@ -1,19 +1,21 @@
 const { get } = require('lodash')
 
+const parkingType = 'parking'
+
 function getSectionAndFloorByUnitName (property, unitName, unitType) {
     const res = {
         sectionName: null,
         sectionType: null,
         floorName: null,
     }
     if (unitName && unitType) {
-        const sections = get(property, ['map', `${unitType !== 'parking' ? 'sections' : 'parking'}`], [])
+        const sections = get(property, ['map', `${unitType !== parkingType ? 'sections' : parkingType}`], [])
```

Comment:
`property.map` is not actually unit type. It is another block and should be get from another constant (in case of it will be changed in the future)

Reply:
You can use `BuildingMap` type `@app/condo/schema` for that